llp.sh

```diff
@@ -0,0 +1,3 @@
+# TODO: avoid needing this
+
+export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$(python -c 'import sysconfig; print(sysconfig.get_config_var("LIBDIR"))')
```

Comment:
I don't understand what this is for

Reply:
i don't remember either...  and it was in my op todo list to look at it.  let's give it a try without.

```suggestion
#export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$(python -c 'import sysconfig; print(sysconfig.get_config_var("LIBDIR"))')
```